src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -302,12 +312,11 @@ class InvoiceForm extends Component {
                       </Col>
                       <Col data-test-col-accounting-code xs={3}>
                         <FieldSelection
-                          dataOptions={[]}
-                          disabled
+                          dataOptions={getCodeOptions(vendorCodes)}
                           label={<FormattedMessage id="ui-invoice.invoice.accountingCode" />}
                           name="accountingCode"
                           required
-                        // validate={validateRequired}
+                          // validate={validateRequired}
```

Comment:
this change doesn't make too much sense. Removing would make more sense, or no changes to this line

Reply:
Let's left this line as is. For now `accounting code` is not required field in the organization form